docs/swagger/swagger.yaml

```diff
@@ -0,0 +1,43 @@
+basePath: /v2
+definitions:
+  items.Response:
+    properties:
+      ID:
+        type: integer
+    type: object
+host: petstore.swagger.io
+info:
+  contact:
+    email: support@swagger.io
+    name: API Support
+    url: http://www.swagger.io/support
+  description: This is a sample server Petstore server.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  termsOfService: http://swagger.io/terms/
+  title: Swagger Example API
+  version: "1.0"
+paths:
```

Comment:
currently repeating stuff from the service file. not DRY.

Reply:
This file is auto-generated.

Reply:
So should it really be committed? A risk to have inconsistent files if not regenerated when quickly changing a service.